addons/product/views/product_views.xml

```diff
@@ -75,6 +75,7 @@
                                 <group name="group_general">
                                     <field name="active" invisible="1"/>
                                     <field name="type" widget="radio" options="{'horizontal': True}"/>
+                                    <field name="service_tracking" invisible="1 or type != 'service'"/>
```

Comment:
Mmmh, as it triggers nothing per se, shouldn't it be invisible here and set visible in sale and event_product ?

cc @Feyensv

Reply:
It is always invisible with the `1`, and made visible by the modules that extend the selection choices (by removing the 1).